Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -195,7 +201,7 @@ extension NewsTableViewController: UITableViewDataSource {
         }
 
         cell.reactionsCollectionView.isHidden = !viewModel.shouldShowReactions
-        cell.reactionsHeightConstraint.constant = cell.reactionsCollectionView.isHidden ? 0 : 30
+        cell.reactionsHeightConstraint.constant = cell.reactionsCollectionView.isHidden ? 0 : AppearanceHelper.shared.reactionCellHeight
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/5e46f122553d1ef99cf3d6085a26f0b620aa295e/Headlines/src/Controllers/NewsTableViewController.swift#L204"></span>Line should be 120 characters or less: currently 136 characters<br />
<code>line_length</code> <code>NewsTableViewController.swift:204</code></td>
    </tr>
  </tbody>
</table>